### DISCUSSION TITLE
Linking gismo to another library that also use Eigen leads to serious errors.

### DISCUSSION THREAD

**Discussion by Skygogogogo** [gismo/gismo#602] [Q&A]
After compiling and installing gismo, linking to another library that also use Eigen leads to
/usr/include/gismo/gsMatrix/Adjugate.h:160:33: error: no ‘const Eigen::internal::adjugate_impl Eigen::MatrixBase::adjugate() const’ member function declared in class ‘Eigen::MatrixBase’
MatrixBase::adjugate() const
/usr/include/gismo/gsMatrix/Adjugate.h:167:50: error: no ‘void Eigen::MatrixBase::adjugateInPlace()’ member function declared in class ‘Eigen::MatrixBase’
inline void MatrixBase::adjugateInPlace()
some many errors that are not declared.

**Comment by Skygogogogo** [comment]
I have tried to make another depend not the copy of Eigen in /path/to/eigen but to gismo's Eigen in /path/to/gismo/external. But no effect, have same errors.

**Reply by hverhelst** [reply]
Is this still relevant?

**Comment by dominik-mokris-mtu** [comment]
I think this is pretty much a duplicate of #566 , which has been resolved. @Skygogogogo re-open, please, when still relevant.
